cmd/gaia/app/export.go

```diff
@@ -171,6 +167,21 @@ func (app *GaiaApp) prepForZeroHeightGenesis(ctx sdk.Context, jailWhiteList []st
 
 	_ = app.stakingKeeper.ApplyAndReturnValidatorSetUpdates(ctx)
 
+	/* Handle fee distribution state - part 2 */
+
+	//reinitialize all distribution delegations
+	for _, del := range dels {
+		app.distrKeeper.Hooks().AfterDelegationModified(ctx, del.DelegatorAddress, del.ValidatorAddress)
+		//si := app.distrKeeper.GetDelegatorStartingInfo(ctx, del.ValidatorAddress, del.DelegatorAddress)
```

Comment:
Why do we have all this commented out code?

Reply:
I'm in the process of debugging https://github.com/cosmos/cosmos-sdk/pull/3656#issuecomment-476332070